pages/index.vue

```diff
@@ -7,7 +7,7 @@
         <Bulletin />
         <WhyJoin id="about" />
         <Events id="events" :items="events" />
-        <FAQ v-if="FAQFlag" id="faq" :items="FAQs" />
+        <FAQ v-if="faqFlag" id="faq" :items="FAQs" />
```

Comment:
Where does this one come from?

Reply:
https://github.com/nwplus/LHD2020/blob/7c57dc460496b82e279475305ef97db09f906893/pages/index.vue#L68

Reply:
Gotcha